### DISCUSSION TITLE
Harmonyos NEXT support

### DISCUSSION THREAD

**Discussion by ccy-213** [tauri-apps/tauri#11715] [General]
Will Tauri eventually supporting Harmonyos NEXT that no longer dependant on android?

**Comment by FabianLars** [comment]
Never say never, but we currently don't have any explicit plans, instead we're focusing on making android work well first. That said, we would not turn down community involvement.

This is our tracking issue btw: https://github.com/tauri-apps/tauri/issues/7287

**Reply by ccy-213** [reply]
Thank you very much

**Comment by AkarinLiu** [comment]
Huawei Release base of HarmonyOS NEXT PC,Is there a clear timeline supporting this platform?

**Reply by FabianLars** [reply]
No. There won't be any progress including planning until we find a contributor or maintainer for this. The current team can't handle this (no time, no interest, no existing knowledge, etc).

**Comment by neoliuhua** [comment]
Many people need tauri to support HarmonyOS PC and mobile platform.

**Comment by vrmaurice** [comment]
I'm just here to support the idea. While right now it isn't a very major platform globally, it is quickly gaining popularity in China, which is definitely not a small market for apps. It won't be long until they start rolling it out worldwide, even on laptops, so this is definitely something to look into.
